docker-compose.yml

```diff
@@ -0,0 +1,821 @@
+version: "3.7"
+services:
+  db:
+    image: mysql:5.7.40
```

Comment:
This version of mysql is not ideal, but I wanted to keep parity with what I assumed the original cluster environment was. Anything beyond 5.7 is requiring sha2 hashing, and I spent many hours trying to figure it out but was ultimately successful. Alternatively, we could use MariaDB instead which I do not think has that imposed requirement.

Reply:
MariaDB latest works absolutely fine, Highly recommend switching to that route. Ugaris is running on MARIADB_VERSION	1:11.1.2+maria~ubu2204 and so far have not run into any kind of issues.